src/Decorations.ts

```diff
@@ -18,16 +19,16 @@ export default class Decorations {
   decorations: NamedDecoration[];
   decorationMap: DecorationMap;
 
-  constructor() {
-    const fontWidthPx = 8.04;
+  constructor(fontSize: FontSize) {
+    console.log(fontSize)
     const iconWidthPx = 4;
-    const iconWidth = `${iconWidthPx}px`;
-    const iconHeight = "1.7em";
+    const iconPadding = 0;
+    const iconWidth = `${iconWidthPx}`;
+    const iconHeight = `${fontSize.fontHeight + iconWidthPx + iconPadding}px`;
```

Comment:
Why add `iconWidthPx` for the height?

Reply:
For the dot its the height as well. We have to look more into that tomorrow.